softgroup/services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/LoginHandler.java

```diff
@@ -0,0 +1,28 @@
+package com.softgroup.authorithation.impl.handler;
+
+import com.softgroup.authorization.api.message.LoginRequest;
+import com.softgroup.authorization.api.message.LoginResponse;
+import com.softgroup.authorization.api.router.AuthorizationRequestHandler;
+import com.softgroup.common.protocol.ProtocolBeansFactory;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.protocol.ResponseStatus;
+import com.softgroup.common.router.api.AbstractRequestHandler;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by nikmlk on 24.02.17.
+ */
+
+@Component
+public class LoginHandler<T extends LoginRequest, R extends LoginResponse> extends AbstractRequestHandler<T, R> implements AuthorizationRequestHandler {
+    @Override
+    public String getName() {
+        return "login";
+    }
+
+    @Override
+    protected Response<R> doHandle(Request<T> msg) {
+        return ProtocolBeansFactory.getResponse(msg, null, ResponseStatus.NOT_IMPLEMENTED);
```

Comment:
@melnyknikolay Maybe it's better to make one more method in factory for error responses ?